Tasks/AzureAppServiceManage/operations/KuduServiceUtils.ts

```diff
@@ -53,8 +54,8 @@ export class KuduServiceUtils {
 
         for(var extensionID of extensionList) {
             var siteExtensionDetails = null;
-            if(siteExtensionMap[extensionID]) {
-                siteExtensionDetails = siteExtensionMap[extensionID];
+            if(siteExtensionMap[extensionID] || (extensionID.startsWith('python') && siteExtensionMap[pythonExtensionPrefix + extensionID])) {
```

Comment:
Should we do case insensitive check?
The same applies for _replace_ call below

Reply:
case insensitive check is not required here, as it worked for early scenarios.
Hpwever, When extension is made dynamic list, it will be handled.